### PR TITLE
WIP VSS public docs

### PR DESCRIPTION
**Pull request by tobru**
This brings the VSHN product documentation to the KB.

It's a WIP, do _not_ merge yet

### CODE REVIEW

File: docs/modules/PRODUCT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= VSHN Product Overview
+
+xref:vss_intro.adoc[VSHN Syn Support]::
+
+Taking care of your Kubernetes Clusters and applications running on them.
+
+xref:managed_server_intro.adoc[VSHN Managed Server]::
+
+Classiv VM managed services
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/index.adoc]: `s/Classiv/Classic/`

File: docs/modules/PRODUCT/pages/vss_intro.adoc [hunk 2]
```diff
@@ -0,0 +1,114 @@
+= Introduction
+
+TIP: This document is currently in public beta and being worked on actively
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_intro.adoc]: Maybe `WARNING` is more appropriate here.

File: docs/modules/PRODUCT/pages/vss_intro.adoc [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Backup::
+
+Regular backup of cluster configuration including monitoring of success.
+
+Monitoring and Alerting::
+
+Monitoring of cluster and system services functionality and alert handling according to SLA. Alert rules and thresholds are continuously assesed and regularly maintained.
++
+NOTE: This includes the Kubernetes cluster and the Project Syn provided tooling, no customer application and services running on the cluster.
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_intro.adoc]: Maybe use ` IMPORTANT:`

File: docs/modules/PRODUCT/pages/vss_intro.adoc [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Keeping an overview of all the Kubernetes clusters, their versions, locations and other important information is provided by Lieutenant and available in the VSHN Portal.
+
+=== VSHN Support and Services
+
+Alert handling::
+
+* Reacting to alerts and handle them according to a specified SLA
+* 24/7 operations (with SLA)
+* Continuous improvement of Alertrules based and day-to-day experience
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_intro.adoc]: `s/based and day-to-day/based on day-to-day/`

File: docs/modules/PRODUCT/pages/vss_price_model.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
+= Price Model
+
+TIP: This document is currently in public beta and being worked on actively
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_price_model.adoc]: `WARNING` might be more appropriate here too.

File: docs/modules/PRODUCT/pages/vss_support_levels.adoc [hunk 6]
```diff
@@ -0,0 +1,53 @@
+= Support Levels
+
+TIP: This document is currently in public beta and being worked on actively
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_support_levels.adoc]: `WARNING` might be more appropriate here too.

File: docs/modules/PRODUCT/pages/vss_support_matrix.adoc [hunk 7]
```diff
@@ -0,0 +1,158 @@
+= Support Matrix
+
+TIP: This document is currently in public beta and being worked on actively
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_support_matrix.adoc]: `WARNING` might be more appropriate here too.

File: docs/modules/PRODUCT/pages/vss_support_matrix.adoc [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+The tables list the officially and by default supported clouds and main features. If a cloud or a feature / combination is not listed, it's not supported by VSHN by default.
+
+* *Provisioning*: How the cluster gets provisioned / configured / customized
+* *Autoscaling*: Is support for autoscaling available or not
+* *Application HA*: Is access to applications available in high-availability? This means: HS Load-balancer and IP HA
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_support_matrix.adoc]: `s/HS/HA/`

File: docs/modules/PRODUCT/pages/vss_vcsp.adoc [hunk 9]
```diff
@@ -0,0 +1,55 @@
+= VSHN Certified Service Provider (VCSP)
+
+TIP: This document is currently in public beta and being worked on actively
```
Comment (corvus-ch) [inline docs/modules/PRODUCT/pages/vss_vcsp.adoc]: Probably also `WARNING`.

**Review by corvus-ch — CHANGES_REQUESTED** [review]

**Comment by tobru** [comment]
Will move to git.vshn.net

**Pull request closed without merging**